### DISCUSSION TITLE
Github Foundation Certification issue

### DISCUSSION THREAD

**Discussion by adityak-cloud** [community/community#144846] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone, 

Actually I have facing one challenges in GitHub foundation certification exam actually I have booking my exam through PSI portal , But there I couldn't found any of mode ex : online & Offline ; and couldn't able to select preferred date &  time slot and major issue is I couldn't apply Voucher in voucher option because there is Grey it means block I couldn't remove and apply any voucher. 

So I have tried to contact PSI exam but they are saying contact to GitHub I have raised 2 ticket no one responsive. and my organisation pay for voucher and its 7 day only valid.

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @adityakanojiya6392!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

**Comment by venkat3990** [comment]
I have cleared the exam on 30/ Jan and it is showing as pass in PSI exam portal, but yet to receive any certificate from GitHub. Anybody else faced this issue ? How can we mitigate this issue.

**Comment by subhagitpom** [comment]
Same here

**Comment by Amanda-Hansamali** [comment]
1. Try Booking from a Different Browser or Device
Use Google Chrome or Mozilla Firefox (preferably in Incognito Mode).
Try a different device (laptop instead of mobile, or vice versa).
2. Verify Your Voucher Code
Double-check if the voucher code is valid and correctly formatted.
Ensure it is not expired and is applicable for your region/country.
 3. Contact GitHub Certification Support (Again)
Since PSI redirected you to GitHub, try these official channels:

GitHub Certification Support Form:
👉 https://certification.github.com/contact

**Comment by james9832** [comment]
If you're facing issues with booking your GitHub Foundation certification exam on the PSI portal, try clearing your browser cache or using a different browser. Contact PSI support for assistance with scheduling, mode selection, and applying your voucher. [Get more info](https://apkberg.com/)!
